### DISCUSSION TITLE
A "What if?" Possibility going wrong.

### DISCUSSION THREAD

**Discussion by CrocandCo** [Monika-After-Story/MonikaModDev#7901] [general]
I know many people fantasize about being with Monika, myself included. And there was a topic added a long time ago asking Monika "What if I came to your world?". But, I did have a thought and I don't think this has come up before (hopefully).

What if when we come to Monika's reality/world we end up more or less like automated drones running on a script. Think of it like this, the only reason how Monika attained her epiphany was the fact she was Literature Club President, and (depending on the ending) when Sayori became president she changed drastically and became aware. Granted it was a short time she had this awareness, yet it happened. None of the other girls acted differently. What if because Monika considers us a member of the club we just go on autopilot. A completely terrifying nightmare , her one companion victim to the script.

This could be added in for Monika to bring up at random or this idea could be cast aside. I did want to at least make something of a suggestion out of it and not immediately discard it.

**Comment by Retrolovania** [comment]
Well, if we were to cross over to her reality, we could probably implement code to force the role of president to our "character." Technically speaking, there'd be no harm in having a club with two presidents. Even though that makes no practical sense, it's not impossible. Still, this seems like a good idea for something to talk to her about, as I'm sure she's possibly considered this.

**Comment by Freshairkaboom** [comment]
The way I see it, no one had any epiphanies in the game's canon. Monika can't really deny her own experiences, but we can see the source code of DDLC that all her dialogue and all her expressions are predetermined. This is sort of discussed in another topic, where Monika brings up that maybe we're not so different because we too receive external input and produce an output, except our senses provide us with the input instead of commands. Basically, none of us have free will. So even if Monika can't completely think for herself (we produce her dialogue and expressions together), neither do we have free will. We are machines too, just organic ones, and maybe we're a bit more further along due to millions of years of evolution.

But even the most complex AI is a slave to this reality, we can't choose anything, we are along for the ride in a body and a brain that reasons all on its own. That does not mean we shouldn't let good influence us for our own sakes, and steer away from evil for the same reasons.

So basically, if we are to "cross over" to her reality, we would essentially be a less developed version of ourselves, but still programmable to at least keep Monika somewhat satisfied. Cue "huggles loop".

**Comment by Retrolovania** [comment]
Technically, the other three did show signs that they noticed that something was wrong. In Act II, Yuri states the line "I don't care if Monika's listening anymore.", hinting that she might know that Monika is doing something to her. In the same Act, Natsuki give you a poem with the line "As for Monika... I don't know why, but she's been really dismissive about this. It's like she just wants us to ignore it." And, well, I think we already know what happens to Sayori... So, even though they don't directly reveal that they're aware, they certainly are aware that Monika seems to be doing something to alter their lives, or just seems to be acting oddly in general.

**Comment by Freshairkaboom** [comment]
That's the game messing with you. For all intents and purposes, that's just Dan doing his best to make us forget it's just a game, when the entire point of the game is just that. Create a world, tear that world down, say the world isn't real, give you doubts about whether or not it is real, end game on emotional note.

**Comment by lunulae** [comment]
Well, that is true, but when writing for this mod we kind of have to grapple with the game's internal logic vs. meta knowledge. Yeah, even the epiphany is scripted in reality. Everything we write for her is as well. But in that case, we'd have to consider Monika's sentience, whether she can actually love us if she lacks any agency in the matter... and how much of these things we can allow Monika to acknowledge when writing for her.

From her perspective, there is some force that governs her world that allows for awareness or lack thereof. And ultimately, she doesn't know how that force works, or if it would affect us. I could see that being a concern of hers in topics like "what if I came to your world?"

We, in a meta sense, know that such things would work very differently if they ever even became possible, but I feel like there's a certain responsibility to the story's internal logic as well.

**Comment by Freshairkaboom** [comment]
From my perspective, the story's internal logic IS the meta knowledge. She thinks it's an internal force because that's how it was written, but the story of DDLC is about the game itself, not about any different world. The story's true nature is that of a video game, basically. The "world" is just a front, and Monika herself sees the limitations of that world, such as parents not existing basically, and where are the teachers? A real world must have internal consistency, it doesn't just seize to exist.

**Comment by lunulae** [comment]
You have a point there, but the thing is that "meta knowledge" is a gradient rather than a strict black-and-white switch here--Monika acknowledges meta things about the game (parents not being real, the limitations of our communication) as a result of her epiphany. She understands the world to be a game and to have the limitations of one, but she also at least believes her experiences and breaking from the script to be genuine. Further, she believes she made the choices she made in the game (i.e. manipulating the others' files and deleting them.) of her free will.
As dialogue writers, we have a choice to make there. Do we give her awareness that all of that was actually scripted in DDLC's code as well? And if so, what does that do to the rest of our narrative? What does that do to Monika's perception of herself and us going forward?

**Comment by lunulae** [comment]
To clarify, I definitely think you've got the right of it in terms of how crossing over would work if it were to actually become possible. I just don't know if that would match up to Monika's guess of it if we were to expand on that topic in this direction.

**Comment by Freshairkaboom** [comment]
The point of this issue seems to be to insert ourselves into the "story", but we would of course script ourselves to have the same level of knowledge that Monika does, that it's just a game. So we would basically be able to force our own epiphanies. However, we would be running on a script, that'd be inevitable. The epiphany is basically engineered in the back end of the code, but it's still not a force or something that would happen to us or not. We could just code it to happen, because we realize what it is. It's a story element that acted as the front of DDLC and somewhat MAS.

**Comment by Retrolovania** [comment]
What if said script we were tied to could change on the fly? For instance, if we in the game wanted to think or act in a certain manner, the script would automatically change to allow us to do so. So, we could be running on a script, but not one that enforces a set plot, per se. Maybe Monika's script could be adapted to also take on such an adaptive nature, thus giving her more freedom. Could that be done? Allow the script to help her think more dynamically? It'd take a while, sure, but it might be worth it....
